UnitTestProject/TrainerTest.cs

```diff
@@ -4,9 +4,326 @@
 using PokemonUnity.Pokemon;
 using PokemonUnity.Attack;
 using PokemonUnity.Item;
+using PokemonUnity.Saving;
+using PokemonUnity.Saving.Location;
+using System.Collections.Generic;
 
 namespace Tests
 {
+    [TestClass] //ToDo: Move Test into a PlayerTest.cs
+    public class PlayerTest
```

Comment:
You didn't do a fetch before posting back to server. Can you refresh your branch and recommit it again please?

Reply:
Sorry, the changes I was thinking seem to still be active in pull request. I thought they were acknowledged already. That was my bad.